src/server/internalServer.cpp

```diff
@@ -675,6 +675,13 @@ std::unique_ptr<Response> InternalServer::handle_search(const RequestContext& re
     printf("** running handle_search\n");
   }
 
+  try {
+    if (request.get_url_part(1) == "searchdescription.xml") {
```

Comment:
This contradicts the commit title. I am 99% confident that you will change the commit message :)

Reply:
Done

Reply:
This was not fully fixed (but one has to be an attentive reader to notice the remaining difference) :smiley:

Reply:
fxied